src/main/java/edu/cornell/kfs/pdp/batch/service/impl/CuExtractPaymentServiceImpl.java

```diff
@@ -112,9 +111,7 @@ public void extractAchPayments() {
     }
 
     /**
-     * Overridden to call a CU-specific version of the parameter-checking method to determine
-     * whether or not to use the ISO 20022 format, as well as to allow for generating
-     * the proprietary XML when the ISO 20022 format is enabled.
+     * Overridden and Cornell customized to not generate proprietary check format.
      */
     @Override
     public void extractChecks() {
```

Comment:
maybe we should  remove this customized function.
In base code, when shouldUseIso20022Format() is false, extractChecksToProprietaryFormat() is called. Since we are Iso20022 format, maybe we don't need this customized function?

@cah292  since you are closest to this, do you have an opinion?

Reply:
Yes, I agree that the `extractChecks()` override and most of the related copied private methods (like `extractChecksToProprietaryFormat()`) can be removed as well. We may still need a base-code-equivalent version of `shouldUseIso20022Format()` if we want to keep the ACH-related customization intact for now. (We'll eventually need to remove the ACH-side proprietary and/or FastTrack customizations, but that could be done at a future date if needed.)

Also, just to be clear, we're still sending the check files to PrintBoss, but they're now processing the ISO 20022 check files instead of the KFS proprietary check files. This PR's description should be updated to reflect that.